### ISSUE TITLE
The Figure 2 in the paper is wrong？

### ISSUE THREAD

**Issue by deepConnectionism** [princeton-vl/DPVO#75]
Figure 2: Feature and Patch Extraction (Sec. 3). Residual networks extract 1) a context feature map at  1/4 resolution and 2) a 2-level pyramid of matching features at  1/4 and 1/8 resolution. Many  patches are cropped from this feature map at random pixel coordinates using bilinear sampling.

1/8 resolution not match in the code, It should be 1/16 resolution?

Assuming the input image is 640, the two feature maps should be 160 and 40.

**Comment by lahavlipson** [comment]
Yes, the lower resolution feature map is at 1/16th the image resolution, not 1/8th
